pages/common/ffmpeg.md

```diff
@@ -4,16 +4,20 @@
 
 - Extract the sound from a video and save it as MP3:
 
-`ffmpeg -i {{video_filename}} -vn -ar 44100 -ac 2 -ab 192 -f mp3 {{sound.mp3}}`
+`ffmpeg -i {{path/to/video}} -vn -ar 44100 -ac 2 -ab 192 -f mp3 {{sound}}.mp3`
 
 - Convert frames from a video or GIF into individual numbered images:
 
-`ffmpeg -i {{video_or_gif_filename}} {{image%d.png}}`
+`ffmpeg -i {{path/to/video|gif}} {{image%d.png}}`
 
 - Combine numbered images (image1.jpg, image2.jpg, etc) into a video or GIF:
 
-`ffmpeg -f image2 -i {{image%d.jpg}} {{video.mpg_or_video.gif}}`
+`ffmpeg -i {{path/to/image%d.jpg}} -f image2 {{video.mpg|video.gif}}`
```

Comment:
The "numbered images (image1.jpg, **image2**.jpg, etc)" part is confusing regarding `-f image2`. Perhaps we could use a different prefix instead? Say, `frame_%d.png` (with an underscore to make the `%d` more visible, and yeah, I also would suggest using png rather than jpg just to encourage lossless formats).

Reply:
Good point.

Regarding png/jpg, it actually depends on your use case. Using jpgs for real life photographs makes sense, otherwise you will blow up storage if you save them in pngs. pngs are more suitable for line drawings, clips and gifs. So, I would suggest keeping jpg as is.

Reply:
Yeah but are jpgs actually used that commonly for frames that can be assembled into meaningful videos? In my experience such frames tend to be more like (temporary) generated images rather than actual photographs, and the final video is then compressed in order to take better advantage of the temporal coherence between frames.

I mean, I know that nowadays there are many cameras with the continuous shooting / sport mode feature, but I wonder how likely it is that people take such sequences only to merge them later as a video (as opposed to actually shooting video in the first place).

But of course, this is a minor point. Let me know what you think.

Reply:
> Yeah but are jpgs actually used that commonly for frames that can be assembled into meaningful videos? 

Yes, I believe so. I have used the exact same thing to create a gif from a series of images. And I had to use jpg images.

Reply:
Ok :+1: